Packs/AWS-CloudTrail/Integrations/AWS-CloudTrail/AWS-CloudTrail.py

```diff
@@ -102,10 +103,10 @@ def aws_session(service='cloudtrail', region=None, roleArn=None, roleSessionName
     return client
 
 
-def handle_returning_date_to_string(date_obj: datetime | str) -> str:
+def handle_returning_date_to_string(date_obj: datetime | str | None) -> str:
     """Gets date object to string"""
-    # if the returning date is a string leave it as is.
-    if isinstance(date_obj, str):
+    # if the returning date is a string or None, leave it as is.
+    if date_obj is None or isinstance(date_obj, str):
```

Comment:
Something is odd about this logic, if date_obj is None and we return it then the function should support None return (in that case let's also make sure it doesn't break any b/c).

Reply:
I would remove the None option from the argument anotation.